### PR TITLE
Adding cloning instructions

### PR DESCRIPTION
**Pull request by bovorasr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,37 @@ with the following base:
     - Catch as many errors as possible at compile vs runtime
     - Scale to full teams of developers
 - React + Redux
-- Webpack
\ No newline at end of file
+- Webpack
+
+
+### How to start a new project ###
+
```
Comment (bovorasr) [inline README.md:14]: Hey hey testing comments

**Comment by bovorasr** [comment]
Let's talk about stuff (testing out pull request features on github)

**Comment by wabbottRAL** [comment]
This seems legit.

**Comment by bovorasr** [comment]
@chriswxyz did you get notified of pull request? I'm more curious about how this works for the pull request approver than anxious about getting the updated README.md in :)

**Comment by chriswxyz** [comment]
I do get notifications, but I think they go to my github email, not work 😞

**Pull request merged**